page-objects/administration.js

```diff
@@ -13,8 +13,9 @@ let newPassword = "Schulcloud1!"
 module.exports = {
 goToAdministration: function() {
     let url = Admin.urlAdministration;
-    return helpers.loadPage(url, 20);
+    return elementHelpers.loadPage(url, 10);
```

Comment:
is the number change wanted here?

Reply:
Das ist unabsichtlich passiert. Danke, dass du so genau geschaut hast! Ich habe es im nächsten Branch zurückgesetzt.